SETUP.md

```diff
@@ -1,101 +1,43 @@
-  - Open command prompt (Open Start Menu then type <code>cmd</code> and press [Enter])</li>
-  - Type the following line into the command prompt window exactly as shown:
-    `setx HOME "%USERPROFILE%"`
-  - Press [Enter], you should see `SUCCESS: Specified value was saved.`
-  - Quit command prompt by typing `exit` then pressing [Enter]
-
-This will provide you with Bash in the Git Bash program.
+Windows does not come with an SSH client. You will need to download one to use during the workshop.  A popular one is called PuTTY. To get PuTTY follow these steps.
+
+1. Download PuTTY from https://the.earth.li/~sgtatham/putty/latest/w32/putty.exe</a>.
+
+2. Place the downloaded file on your desktop for easy access.
 
 ### macOS
 
-The default shell in all versions of macOS is Bash, so no need to install 
-anything. You access Bash from the Terminal (found in `/Applications/Utilities`).
-See the Git installation <a href="https://www.youtube.com/watch?v=9LQhwETCdwY ">
-video tutorial</a> for an example on how to open the Terminal. You may want to 
-keep Terminal in your dock for this workshop.
+On macOS there is a built in SSH client you can access from your Terminal. You can find the Terminal by opening your Applications folder, then opening the Utilities folder. The Terminal application will be amongst the icons there. You may want to add this to your dock.
 
 ### Linux
 
-The default shell is usually Bash, but if your machine is set up differently you 
-can run it by opening a terminal and typing `bash`. There is no need to install 
-anything.
+On Linux there is a built in SSH client you can access from your Terminal. Linux has many ways to access the terminal.  If your distribution has a search function (like UBUNTU) look for one of these keywords: terminal, xterm, uxterm. 
 
 ## Text Editor
 
-When you're writing code, it's nice to have a text editor that is optimized for 
-writing code, with features like automatic color-coding of key words.  The 
-default text editor on macOS and Linux is usually set to Vim, which is not 
-famous for being intuitive. if you accidentally find yourself stuck in it, try 
-typing the escape key, followed by `:q!` (colon, lower-case 'q', exclamation 
-mark), then hitting Return to return to the shell.
+A text editor may be a helpful tool when you are working with SSH. Any editor you can write text on will do. Some examples are below.
 
 ### Windows
-
-<a href="https://www.youtube.com/watch?v=339AEqk9c-8">Video Tutorial</a>
-
-nano is a basic editor and the default that instructors use in the workshop.
-To install it, download the <a href="https://github.com/swcarpentry/windows-installer/releases/download/v0.3/SWCarpentryInstaller.exe">
-Software Carpentry Windows installer</a> and double click on the file to run it.
-This installer requires an active internet connection.
-
-Others editors that you can use are:
-
+- NotePad (Pre-installed)
```

Comment:
Thought to use Sublime for all, but agree with using pre-installed software as much as possible

Reply:
I just figured its a scratch pad,  i mean i do not directly use it in anyway in my workshops and I don't think you do in yours.